### ISSUE TITLE
[1.3-rc.4] Error when loading node-sass using 'import' instead of 'Npm.require' in build plugin on Windows

### ISSUE THREAD

**Issue by nathantreid** [meteor/meteor#6549]
I'm running Windows 10 x64 with node 0.10.40 32-bit and npm 1.4.28. 
When loading node-sass in a build plugin, everything works fine if I use `const sass = Npm.require('node-sass');`.
However, if I replace that with `import sass from 'node-sass';`, I get the following error: 

```
While loading plugin `scss-build-plugin` from package `sass-test`:
   packages/scss-build-plugin/sass-test.js:11:7: The `libsass` binding was not found in
   \node_modules\meteor\scss-build-plugin\node_modules\node-sass\vendor\win32-ia32-11\binding.node
   This usually happens because your node version has changed.
   Run `npm rebuild node-sass` to build the binding for your current node version.
   at Object.sass.getBinaryPath (packages/scss-build-plugin/sass-test.js:11:7)
   at meteorInstall.node_modules.meteor.scss-build-plugin.node_modules.node-sass.lib.index.js
   (packages/scss-build-plugin/sass-test.js:11:7)
   at fileEvaluate (<runJavaScript-96>:158:9)
   at require (<runJavaScript-96>:92:16)
   at meteorInstall.node_modules.meteor.scss-build-plugin.sass-test.js
   (packages/scss-build-plugin/sass-test.js:1:1)
   at fileEvaluate (<runJavaScript-96>:158:9)
   at require (<runJavaScript-96>:92:16)
   at packages/scss-build-plugin/sass-test.js:11:7
   at packages/scss-build-plugin/sass-test.js:11:7
```
The binding does exist, but there is no path 'node_modules\meteor'; if I set the path manually via an environment variable, then it works again: 
`set SASS_BINARY_PATH=H:\projects\node-sass-test-1.3\packages\sass-test\.npm\plugin\scss-build-plugin\node_modules\node-sass\vendor\win32-ia32-11\binding.node`).

For testing, I also loaded node-sass on the server-side via an import statement (not as a build plugin but in regular server-side code) and that works fine, so the issue appears to be how imports are handled by the build plugin system.

Sample Repo: https://github.com/nathantreid/meteor-node-sass-test-1.3

**Comment by nathantreid** [comment]
Meteor is duplicating the build plugin name in the path: 
.meteor\local\isopacks\sass-test\plugin.scss-build-plugin.os.windows.x86_32\npm\node_modules\meteor\ **scss-build-plugin\scss-build-plugin** \node_modules\node-sass\vendor\win32-ia32-11\binding.node

On a side note, even without the duplication the path is 20 characters longer than it was in 1.2. This is causing me to hit the Windows path limit length much quicker:
1.2: .meteor\local\isopacks\sass-test\plugin.scss-build-plugin.os.windows.x86_32\npm\scss-build-plugin\node_modules\node-sass\vendor\win32-ia32-11\binding.node

1.3: .meteor\local\isopacks\sass-test\plugin.scss-build-plugin.os.windows.x86_32\npm\node_modules\meteor\scss-build-plugin\scss-build-plugin\node_modules\node-sass\vendor\win32-ia32-11\binding.node

**Comment by benjamn** [comment]
Working on this now. Note: this is a cross platform bug, not only a Windows problem.

**Comment by benjamn** [comment]
Fixed in [`1.3-rc.5`](https://github.com/meteor/meteor/issues/6266#issuecomment-199632139), though I recommend updating to [`1.3-rc.6`](https://github.com/meteor/meteor/issues/6266#issuecomment-199903292) to avoid #6555.
